content/en/building/login.md

```diff
@@ -53,11 +21,13 @@ On desktop devices, the user must login again if they close the app tab or brows
 
 Users may log out by going to the options menu available in the top right corner of the app.
 
-{{< see-also page="building/concepts/navigation" >}}
+{{< see-also page="/building/navigation" >}}
 
-{{< figure src="login-mobile.png" link="login-mobile.png" class="left col-3 col-lg-3" >}}
+{{< cards >}}
+    {{< card image="login-mobile.png" link="login-mobile.png"  subtitle="Mobile login - click to zoom" >}}
+    {{< card image="login-desktop.png" link="login-desktop.png"  subtitle="Desktop login - click to zoom" >}}
+{{< /cards >}}
```

Comment:
Sorry for being picky about this 😅 Could you please try the following suggestion?

```suggestion
{{< figure src="login-desktop.png" link="login-desktop.png" class="right col-6 col-lg-8" >}}

{{< cards >}}
{{< figure src="login-mobile.png" link="login-mobile.png" class="right col-6 col-lg-8" >}}
{{< /cards >}}
```

Also, please use the screenshots I generated with the automation test [here](https://drive.google.com/drive/folders/1ILOBL7ZK0sPmWtnElLcYtS22gWTP68a3?usp=drive_link). They follow the predefined sizes, so all screenshots will appear consistently sized. Thanks!

Reply:
Ok - let's go for it!  I've used your suggestion and screenshots in this commit, but since the CSS you cited (`class="right col-6 col-lg-8"`) doesn't exist in Hextra (it's from Docsy), I've removed those.  see fc3cfdfc